processor/queuedprocessor/queued_processor_test.go

```diff
@@ -106,12 +106,8 @@ func TestQueueProcessorHappyPath(t *testing.T) {
 	// Wait until all batches received
 	mockProc.awaitAsyncProcessing()
 
-	if wantBatches != int(mockProc.batchCount) {
-		t.Fatalf("Wanted %d batches, got %d", wantBatches, mockProc.batchCount)
-	}
-	if wantSpans != int(mockProc.spanCount) {
-		t.Fatalf("Wanted %d spans, got %d", wantSpans, mockProc.spanCount)
-	}
+	require.Equal(t, wantBatches, int(mockProc.batchCount), "Incorrect batches count")
+	require.Equal(t, wantSpans, int(mockProc.spanCount), "Incorrect batches spans")
```

Comment:
I think the error message "Incorrect batches count" could be removed as I remember from comments from my previous PR. @pjanotti , right?

Reply:
for the second `require` "Incorrect span count" seems better, that said in general I do prefer require/assert without the message since one typically have to look at the whole test anyway. That said we haven't been enforcing it one way or another, depending on the context I have suggested in the past for the message to be removed.

Reply:
this case in special is pretty straightforward and the message doesn't bring much, require/assert already produces a message for Equal when values do not match.